micropub.php

```diff
@@ -82,6 +82,10 @@ public static function init() {
 		add_action( 'send_headers', array( $cls, 'http_header' ) );
 		add_filter( 'host_meta', array( $cls, 'jrd_links' ) );
 		add_filter( 'webfinger_user_data', array( $cls, 'jrd_links' ) );
+
+		// Post Content Filter
+		add_filter( 'micropub_post_content', array( $cls, 'generate_post_content' ), 10, 2 );
```

Comment:
we should probably use a much higher priority than 10 here, since we always want `generate_post_content` to run first, right? maybe 1 or 0?

Reply:
10 is default. But why should this run before adding properties?

Reply:
so they can see the content this plugin generates! they may want to tweak it instead of replacing it entirely.

Reply:
Good point, but in that case, I'll set the other options to 11 and 12. It would be atypical otherwise.

Reply:
no, i think we want 1 or 0. remember, _anyone_ can use a filter. you can't control other people's plugins and themes. you want to make the normal, default path work for other people, without them doing anything special. :P